packages/website/components/enterpriseTierRequestModal/enterpriseTierRequestModal.js

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useLayoutEffect } from 'react';
+import kwesforms from 'kwesforms';
+
+import Modal from 'modules/zero/components/modal/modal';
+import CloseIcon from 'assets/icons/close';
+import Button from 'components/button/button.js';
+import GradientBackground from 'components/gradientbackground/gradientbackground';
+
+const UserRequestModal = ({ isOpen, onClose }) => {
```

Comment:
I was a little confused about this name. And I see later we import it as `EnterpriseTierRequestModal`. wdyt of calling it that in here too?

Might also be good to have a jsdoc explaining what it's for, that way another dev that finds it can understand if it's a good fit for what they're doing. jsdoc for the two props types could also help avoid bugs with another dev calling it wrong. (I've fixed some website bugs that would have been caught by having prop types in jsdoc)